### PR TITLE
tomato soup helps against babby diseases

### PR DESCRIPTION
**Pull request by SonixApache**
🆑 
 - tweak: Tomato soup now helps against very weak diseases.

### CODE REVIEW

**Comment by GreatBigFailure** [comment]
Food should have little effects like this. This is good.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -3152,6 +3152,31 @@
...
+	reagent_state = REAGENT_STATE_LIQUID
+	nutriment_factor = 5 * REAGENTS_METABOLISM
+	color = "#731008" //rgb: 115, 16, 8
+	density = 0.63
+	specheatcap = 4.21
+	data = list(
+		"threshold" = 10,
+		)
+
+/datum/reagent/tomato_soup/on_mob_life(var/mob/living/M)
```
Comment (ShiftyRail) [inline code/modules/reagents/Chemistry-Reagents.dm]: `/datum/reagent/antipathogenic/tomato_soup/` ?

Comment (SonixApache) [reply code/modules/reagents/Chemistry-Reagents.dm]: hahaha i am dumb

**Review by ghost — APPROVED** [review]

**Comment by FudgePucker** [comment]
Chicken noodle soup when? 10u water, chicken, and spaghetti.

**Pull request merged**